pyproject.toml

```diff
@@ -15,6 +15,9 @@ passlib = "^1.7.4"
 crabatar = "^1.0.3"
 geoip2 = "^4.4.0"
 python-dotenv = "^0.19.1"
+boto3 = "^1.19.7"
+PyMySQL = "^1.0.2"
```

Comment:
What's the reason from switching from Postgres to MySQL?

Reply:
PostgreSQL's `GROUP BY` functions differently from SQLite3 and MySQL in a way that breaks a lot of important queries in Crabber (e.g. Notifications). Frankly I don't know how to rewrite them in a way that *works* with PostgreSQL in SQLAlchemy.

More specifically it doesn't allow you to `SELECT` any columns that aren't used in the `GROUP BY` (like `Notification.id` which is required for ORM) and splitting the query into two subqueries still didn't fix anything. I'd be open to switching back if someone has an idea on how to solve this but otherwise MySQL seems to work much better for this particular project.

Reply:
Ah I see